File: explore_json.js
const fs = require('fs');
const data = JSON.parse(fs.readFileSync('C:/Users/HYPE-R/.gemini/antigravity/playground/white-orion/api_response.json', 'utf8'));

console.log("Keys in root:", Object.keys(data));
if (data.dailyActivity && data.dailyActivity.length > 0) {
    console.log("Keys in dailyActivity[0]:", Object.keys(data.dailyActivity[0]));
    console.log("Sample dailyActivity[0]:", JSON.stringify(data.dailyActivity[0], null, 2));
} else {
    console.log("dailyActivity is empty or missing");
}

if (data.template && data.template.length > 0) {
    console.log("Keys in template[0]:", Object.keys(data.template[0]));
}

File: explore_json.py
import json

with open('C:/Users/HYPE-R/.gemini/antigravity/playground/white-orion/api_response.json', 'r', encoding='utf-8') as f:
    data = json.load(f)

print("Keys in root:", list(data.keys()))

if 'dailyActivity' in data and len(data['dailyActivity']) > 0:
    print("Keys in dailyActivity[0]:", list(data['dailyActivity'][0].keys()))
    print("Sample dailyActivity[0]:", json.dumps(data['dailyActivity'][0], indent=2))
else:
    print("dailyActivity is empty or missing")

if 'template' in data and len(data['template']) > 0:
    print("Keys in template[0]:", list(data['template'][0].keys()))
    print("Sample template[0]:", json.dumps(data['template'][0], indent=2))
